modules/creating_a_module.md

```diff
@@ -20,6 +20,8 @@ The [JHipster Fortune module](https://github.com/jdubois/generator-jhipster-fort
 
 It is our sample module that showcases how you can JHipster's variables and functions in order to create your own generator.
 
+Or, you can use the [JHipster module](https://github.com/pascalgrimaud/generator-jhipster-module) to help you to initialize your module.
```

Comment:
And if you prefer this module to be official, in JHipster organization, don't hesitate! Just ask. I can donate this project (I think it's possible, right ?)